### ISSUE TITLE
[4.0.0] indent with flatTernaryExpressions

### ISSUE THREAD

**Issue by mysticatea** [eslint/eslint#8710]
**Tell us about your environment**

* **ESLint Version:** 4.0.0
* **Node Version:** 8.1.0
* **npm Version:** 5.0.3

**What parser (default, Babel-ESLint, etc.) are you using?**

- default

**Please show your full configuration:**

```
{
    "extends": ["mysticatea", "mysticatea/node"],
    "rules": {
        "indent": ["error", 4, {"SwitchCase": 1, "flatTernaryExpressions": true}]
    }
}
```

**What did you do? Please include the actual source code causing the issue.**

- [no-unsupported-features.js#L94-L98](https://github.com/mysticatea/eslint-plugin-node/blob/5634718d468a69e2aea53988341721a37e8fcb4c/lib/rules/no-unsupported-features.js#L94-L98)

```js
    return (
        a == null ? b :
        semver.lt(a, b) ? a :
        /* otherwise */ b
    )
```

**What did you expect to happen?**

No errors.

**What actually happened? Please include the actual, raw output from ESLint.**

```
C:\Users\starc\Documents\GitHub\eslint-plugin-node\lib\rules\no-unsupported-features.js
   96:1  error  Expected indentation of 4 spaces but found 8  indent
   97:1  error  Expected indentation of 4 spaces but found 8  indent
  110:1  error  Expected indentation of 4 spaces but found 8  indent
  111:1  error  Expected indentation of 4 spaces but found 8  indent
```
